scaffoldmaker/meshtypes/meshtype_3d_lens1.py

```diff
@@ -0,0 +1,250 @@
+"""
+Generates a 3-D lens mesh with variable numbers of elements
+around, up the central axis, and radially.
+"""
+
+from __future__ import division
+import math
+from scaffoldmaker.meshtypes.meshtype_3d_solidsphere1 import MeshType_3d_solidsphere1
+from scaffoldmaker.utils.meshrefinement import MeshRefinement
+from scaffoldmaker.utils.zinc_utils import *
+from opencmiss.zinc.element import Element, Elementbasis, Elementfieldtemplate
+from opencmiss.zinc.field import Field
+from opencmiss.zinc.node import Node
+
+class MeshType_3d_lens1:
+    '''
+    classdocs
```

Comment:
Describe class here. Can be same text as at top of file, but more detailed.
Worth mentioning that it is a spherical lens, and that the lens is created by a function morphing a sphere to the lens.